django/publicmapping/run_django.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+python manage.py migrate
```

Comment:
I don't think this is the type of command we want to be running when the container is instantiated. Usually, we try to break out management commands that require database interaction such that they're run after deployments. For example, if we wanted to achieve zero-downtime deployment, and we already had a version of the application running (v1)—the instantiation of a newer version of the same service (v2) could mutate the database in a way that breaks v1 (e.g., dropping a column) before v2 ever becomes health enough to do work.

In this case, I think that moving the invocation of `manage.py migrate` to `scripts/update` would be better.